apps/coscrad-frontend/src/components/resource-info/resource-info.spec.tsx

```diff
@@ -192,4 +195,35 @@ describe.skip('AllResources', () => {
             await waitFor(() => expect(screen.getByTestId('error')).toBeTruthy());
         });
     });
+
+    it('should apply the custom backgroundColor', () => {
```

Comment:
Is this test wrapped in its own describe block? It shouldn't be "floating around" on its own.